roles/api_load/templates/api_load.yml

```diff
@@ -61,21 +61,33 @@ spec:
         command: ["/bin/sh", "-c"]
         args:
           - >
+{% for key, value in workload_args.test_list.items() %}
             ocm-load-test
 {% if workload_args.override is defined %}
             {{ workload_args.override }}
 {% else %}
             --test-id={{ uuid }}
             --gateway-url {{ workload_args.gateway_url }}
             --ocm-token={{ workload_args.ocm_token }}
+{% if value.duration is defined %}
+            --duration={{ value.duration}}
```

Comment:
instead of using if's for these, what about just defaulting the value? ieL
```
--duration={{ value.duration | default(60) }}
```

Reply:
ohhh nice trick!